tests/test_app.py

```diff
@@ -367,7 +367,8 @@ def test_startup_failure(tmpdir, postgres_db, database_name):
     assert code == 4
 
 
-def test_compiler_exception_output(snippetcompiler):
+@pytest.mark.parametrize("cache_cf_files", [True, False])
+def test_compiler_exception_output(snippetcompiler, cache_cf_files):
```

Comment:
I would add a comment to indicate this test case is used to also test the caching and why there is no assertions related to that.  (basic smoke test for argument parsing)